masterlist.yaml

```diff
@@ -16063,7 +16071,9 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/49868/'
         name: 'Imperial City Canal Overhaul'
-    after: [ 'Market District Landscape Fix.esp' ]
+    after:
+      - 'Imperial City Landscape Fix.esp'
+      - 'Market District Landscape Fix.esp'
```

Comment:
The author also recommends loading the plugins early, so these rules can be replaced by placing them in the `Early Loaders` group.